### PR TITLE
parameterise lambda tests

### PR DESCRIPTION
**Pull request by cosmic-byte**
## Description
Reduce cut-and-paste code in .circleci/config.yml lambdas tests by reusing (and parameterizing) a single test

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -100,9 +104,14 @@ jobs:
...
     <<: *test-base
 
-  test-lambdas-indexer:
+  test-lambda:
     docker:
       - image: circleci/python:3.6-jessie
+    description: "Test lambdas"
+    parameters:
+      path:
+        description: "The lambda path"
```
Comment (akarve) [inline .circleci/config.yml:113]:
```suggestion
        description: "lambdas/$path"
```

Comment (cosmic-byte) [reply .circleci/config.yml:115]: I don't think you can reference `parameters.path` inside the description of the parameter definition

Comment (cosmic-byte) [reply .circleci/config.yml:115]: Oh, you mean just a literal string `"lambdas/$path"`

File: .circleci/config.yml [hunk 2]
```diff
@@ -100,67 +104,16 @@ jobs:
...
-  test-lambdas-search:
+  test-lambda:
     docker:
       - image: circleci/python:3.6-jessie
+    description: "Test lambdas"
+    environment:
+      QUILT_DISABLE_USAGE_METRICS: true
+    parameters:
+      path:
+        description: "lambdas/$path"
```
Comment (akarve) [inline .circleci/config.yml:115]: Sorry, here's a better description:
```suggestion
        description: "Relative path to lambda root including setup.py, e.g. 'es/indexer' for lambdas/es/indexer"
```

**Pull request merged**